driver_cpl/cime_config/config_component.xml

```diff
@@ -9,16 +9,13 @@
      should be provided by the driver. This list should follow the same order
      as compset longnames follows -->
 
-  <components>
-    <comp>DRV</comp>
-    <comp>ATM</comp>
-    <comp>LND</comp>
-    <comp>ICE</comp>
-    <comp>OCN</comp>
-    <comp>ROF</comp>
-    <comp>GLC</comp>
-    <comp>WAV</comp>
-  </components>
+  <entry id="COMP_CLASSES">
```

Comment:
Change this structure to be consistant with the rest of the file, this also means that it ends up in env_case.xml and can be retrieved using case.get_value.   Adding a new component should just mean appending this list.